rust/toolchain.bzl

```diff
@@ -3,18 +3,24 @@
 load("//rust/private:common.bzl", "rust_common")
 load("//rust/private:utils.bzl", "dedent", "find_cc_toolchain")
 
-def _make_dota(ctx, file):
-    """Add a symlink for a file that ends in .a, so it can be used as a staticlib.
+def _make_dota(ctx, rlib_file, postfix = ""):
```

Comment:
Could we omit the .rlib/-rlib suffix completely? So `libcore-3e9f2a4814daef68.rlib` will have a symlink named `libcore-3e9f2a4814daef68.a`? I guess we can find a different way to find rust stdlib rlibs in tests (for example by looking at the directory path?)

Reply:
Ideally, yes. AFAIK the reason why the suffix exists today is so that the stdlib_ordering unit test can distinguish between standard and user-defined libs:
https://github.com/bazelbuild/rules_rust/blob/404b84bd6f614cfff19d22de9a984d2f9717ca50/test/unit/stdlib_ordering/stdlib_ordering.bzl#L33

This seems pretty fragile. I'll look into sieving out the standard from user-defined libs in a different way there, so we can drop the prefix here.